File: many_smaller.py
def smaller(arr):
    new_arr = []
    count = 0
    while len(arr) > 0:
        first = arr.pop(0)
        for x in range(len(arr)):
            if first > arr[x]:
                count += 1
        new_arr.append(count)
        count = 0
    return new_arr

File: manySmaller.js
function smaller(nums) {
    let arr = [];
    let count = 0;
    while (nums.length > 0){
      let first = nums.shift();
      for (let i = 0; i < nums.length; i++){
        if (first > nums[i]){
          count++;
        }
      }
      arr.push(count);
      count = 0;
    }
    return arr;
    }